### ISSUE TITLE
Players Coloring

### ISSUE THREAD

**Issue by gustavocdc** [Speedi13/ROP-COMPILER#17]
I'm using only the 'CSGO_MinimapESP.asm' but the hack is coloring every player (teammates and enemies) in Red, can you fix it? I already figured it out by myself how to change the Red to other colors, or comment the code to not apply any colors. But, it would be very nice to have it fixed, so it will color only enemies, or perhaps color enemies with 1 color and teammates with other color. Thanks.

**Comment by Speedi13** [comment]
> but the hack is coloring every player (teammates and enemies) in Red, can you fix it?
No. The code is coloring every player that has not the same team id as the local player. Sometimes it can happen that the team id of the local player changes for some reason.

You could add a second check if the player is same team as local player, set a different color.
